src/System.ServiceModel.Tests.Common/src/System.ServiceModel.Tests.Common.csproj

```diff
@@ -14,9 +14,6 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{E896294A-AB4A-4AF5-A01C-A19E3972EFF9}</ProjectGuid>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(UseFiddler)' == 'true'">
```

Comment:
Note: (no action required) I opened #129 to reenable the Fiddler option using TestProperties